assets/scripts/playlist/playlistTable.jsx

```diff
@@ -189,37 +55,78 @@ var PlaylistTable = React.createClass({
 	}
 });
 
-var PlaylistItems = React.createClass({
+var PlaylistItem = React.createClass({
+	navigateToArtist: function(playList) {
+		store.dispatch({
```

Comment:
We should be using .connect() to dispatch actions, don't assume the `store` object will always be available